spec/controllers/recipes_controller_spec.rb

```diff
@@ -0,0 +1,134 @@
+require 'rails_helper'
+
+describe RecipesController do
+  describe "index" do
+    context "with correct user's token auth" do
+      before do
+        @user = User.create!(user_attributes)
+        auth(@user)
+      end
+      it "returns all user's recipes" do
+        Recipe.create!(recipe_attributes)
+        recipes = @user.recipes.load
+
+        get :index, format: :json
+
+        expect(response.status).to eq 200
+        expect(response.body).to have_content(recipes.to_json)
```

Comment:
Just my opinion, but I prefer having only one `expect` per example, if possible. So I'd refactor this into two examples, like this:

```ruby
it "returns http 200" do
  Recipe.create!(recipe_attributes)
  recipes = @user.recipes.load

  get :index, format: :json
  expect(response.status).to eq 200
end

it "returns all user's recipes" do
  Recipe.create!(recipe_attributes)
  recipes = @user.recipes.load

  get :index, format: :json
  expect(response.body).to have_content(recipes.to_json)
end
```